docs/installation.rst

```diff
@@ -1,21 +1,25 @@
 Download and Installation
 =========================
 
+ABCE works exclusively with python 3!
 
 
 Installation Ubuntu
 -------------------
 
 1. If pip not installed in terminal [#fail]_ ::
 
-    
-    sudo apt-get install python-pip python-dev python-pandas python-setuptools
+    sudo apt-get install python3
+    sudo apt-get install python3-pip python-dev python-pandas python-setuptools
```

Comment:
Travis tests the env when the dependencies are installed via `pip` instead of from `apt-get`. This should have been sufficient with `sudo apt-get install python3 python3-dev python3-pip`, though it is fine as it is.